integration-cli/fixtures/plugin/plugin.go

```diff
@@ -155,6 +155,30 @@ func makePluginBundle(inPath string, opts ...CreateOpt) (io.ReadCloser, error) {
 	if err := os.MkdirAll(filepath.Join(inPath, "rootfs", filepath.Dir(p.Entrypoint[0])), 0755); err != nil {
 		return nil, errors.Wrap(err, "error creating plugin rootfs dir")
 	}
+
+	// Ensure the mount target paths exist
+	for _, m := range p.Mounts {
+		stat, err := os.Stat(*m.Source)
```

Comment:
Janky panicked due to test case where `m.Source` is nil.
https://github.com/moby/moby/blob/8aa694c91ac727351ebb1deaacad62149e984fd6/integration-cli/docker_cli_plugins_test.go#L175

Please check for `m.Source != nil`